clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -355,7 +361,7 @@ private KafkaAdminClient(AdminClientConfig config, String clientId, Time time, M
         this.maxRetries = config.getInt(AdminClientConfig.RETRIES_CONFIG);
         config.logUnused();
         AppInfoParser.registerAppInfo(JMX_PREFIX, clientId);
-        log.debug("Kafka admin client with client id {} created", this.clientId);
+        log.debug("Kafka admin client created");
```

Comment:
Maybe we should say `initialized` instead of `created` for consistency with the consumer.

Reply:
Updated the log.